action_tutorials/README.md

```diff
@@ -0,0 +1,21 @@
+# ROS2 Action Tutorials
+
+This tutorial demonstrates implementing ROS action servers and action clients.
+
+The action server in this case takes in an integer value between 0 and 45 named *order* and returns the Fibonacci sequence, a sequence with the form:
+$$F_0 = 0$$
+$$F_1 = 1$$
+$$F_{order}=F_{order-1} + F_{order-2}$$
+
+The action server calculates each number in the sequence one at a time and returns a partial sequence as feedback at each iteration. 
+If the action is cancelled before the entire sequence is calculated, a partial sequence will be returned.
```

Comment:
I think this sentence could be a bit clearer. It makes it seem as if the partial sequence is returned upon cancellation, but cancellation will simply make the server discontinue its operation on the action request.